### DISCUSSION TITLE
Welcome to Recipe-App Discussions!

### DISCUSSION THREAD

**Discussion by TianMeds** [TianMeds/Recipe-App#1] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Chetangoyal1007** [comment]
i am not able to use google script for email

**Reply by TianMeds** [reply]
Do you mean the message.js file ?

**Comment by preethi8884** [comment]
how to connect to the backend ?? i am not getting
